src/assets/amountInput.tsx

```diff
@@ -4,7 +4,7 @@ import * as React from 'react';
 import { useIMask } from 'react-imask';
 import { BalanceAssets } from 'ui/reducers/updateState';
 import { useAppSelector } from 'ui/store';
-import * as styles from './amountInput.module.css';
+import styles from './amountInput.module.css';
```

Comment:
May be enable `namedExport` for css-loader instead?

Reply:
I don't think this is a good idea. It is better for styles to have one name.

Reply:
It's still one name, it will just export the same thing as before, so these changes would be unnecessary

Reply:
But it's already fixed (:

Reply:
Okay, let's leave it changed then)